VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/individual-participant-status-list/individual-participant-status-list.component.ts

```diff
@@ -89,8 +94,12 @@ export class IndividualParticipantStatusListComponent extends WRParticipantStatu
         if (!endpoint.defence_advocate_username) {
             return false;
         }
-        const requester = this.getConsultationRequester();
-        if (requester.username.toLowerCase() !== endpoint.defence_advocate_username) {
+        // const requester = this.getConsultationRequester();
+        // if (requester.username.toLowerCase() !== endpoint.defence_advocate_username) {
+
+        if (
+            endpoint.defence_advocate_username.toLocaleLowerCase().trim() !== this.adalService.userInfo.userName.toLocaleLowerCase().trim()
```

Comment:
will the current user always be the requester?
what did `getConsultationRequester` previously return?

Reply:
previously it returned Participant from Conference by logged username (username from adalService..), don't know why it's done in such way, as it seems that requester is the current logged user.
If the requester is not a defence advocate, the false should be returned.